python/python/glide/config.py

```diff
@@ -51,6 +51,14 @@ class ProtocolVersion(Enum):
     """
 
 
+class TlsMode(Enum):
+    """
+    Enabled TLS mode, with automatic loading of certificates.
```

Comment:
with automatic loading of certificates. = > with built-in set of CA certificates

Reply:
built-in into what?